### PR TITLE
feat(cli): switch .template to .ejs

### PR DESCRIPTION
**Pull request by virkt25**
Switch `.template` to `.ejs` for CLI template files. Gives us syntax highlighting instead of error in VSCode.

Follow up from https://github.com/strongloop/loopback-next/pull/991

## Checklist

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] Related API Documentation was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `packages/example-*` were updated

### CODE REVIEW

File: packages/cli/generators/project/templates/.prettierignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
 dist
-dist6
```
Comment (bajtos) [inline packages/cli/generators/project/templates/.prettierignore]: I am not a fan of making unrelated changes. Can you at least split the removal of `dist6` entries into a standalone commit please?

File: packages/cli/generators/project/templates/_.gitignore [hunk 2]
```diff
@@ -62,5 +62,3 @@ api-docs/
 
 # Transpiled JavaScript files from Typescript
 dist/
-dist6/
```
Comment (bajtos) [inline packages/cli/generators/project/templates/_.gitignore]: Ditto.

**Comment by virkt25** [comment]
@bajtos Feedback applied. See #998 for template changes unrelated to name.

**Comment by virkt25** [comment]
Current tests assert that most of the files that have been renamed exist in a generated project (with actual name not with `.ejs` and check contents (usually a line not entire file contents). So I believe we are covered. You can find those tests in `packages/cli/test/project.js`, `app.js`, and `extension.js` in same folder as first test file.
